README.md

```diff
@@ -196,29 +193,29 @@ Print the number E to 15 decimal places
 
 Print UNIX time (seconds since Jan 1, 1970, 00:00:00 UTC)
 
-    perl -le 'print time'
+    perl6 -e 'say time'
 
 Print GMT (Greenwich Mean Time) and local computer time
 
-    perl -le 'print scalar gmtime'
-    perl -le 'print scalar localtime'
+    perl6 -MDateTime::TimeZone -e 'say to-timezone("GMT",DateTime.now)'
+    perl6 -e 'say DateTime.now'
 
 Print local computer time in H:M:S format
 
-    perl -le 'print join ":", (localtime)[2,1,0]'
+    perl6 -e 'say DateTime.now.map({$_.hour, $_.minute, $_.second.round}).join(":")'
 
 Print yesterday's date
 
-    perl -MPOSIX -le '@now = localtime; $now[3] -= 1; print scalar localtime mktime @now'
+    perl6 -e 'say DateTime.now.earlier(:1day)'
 
 Print date 14 months, 9 days and 7 seconds ago
 
-    perl -MPOSIX -le '@now = localtime; $now[0] -= 7; $now[4] -= 14; $now[7] -= 9; print scalar localtime mktime @now'
+    perl6 -e 'say DateTime.now.earlier(:14months).earlier(:9days).earlier(:7seconds)'
```

Comment:
Hello, I'm watching this repository on GitHub, so I noticed this pull request. :smiley_cat:

I'm running Perl 6 v6.c. `:14months` gives me a run-time error ("Month out of range"), but this works (not surprisingly):

```perl6
DateTime.now.earlier(:1year).earlier(:2months).earlier(:9days).earlier(:7seconds)
```